### DISCUSSION TITLE
how to upgrade my minio without the Internet, there is only a LAN

### DISCUSSION THREAD

**Discussion by wuyuan0717** [minio/minio#12936] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
without the Internet, there is only a LAN,no mirror server

**Describe the solution you'd like**
hope that `mc admin update` supports reading Minio binary files uploaded to the server to update the version

**Describe alternatives you've considered**
upgrade from Source??

**Comment by harshavardhana** [comment, marked as answer]
Download binary and replace the binary - restart server.

https://min.io/download#/linux
